### ISSUE TITLE
AMR 2.0 support

### ISSUE THREAD

**Issue by danielhers** [ida-szubert/amr_ud#4]
I think these alignments would have better impact with [LDC2017T10](https://catalog.ldc.upenn.edu/LDC2017T10) support. Much of the data is the same as 1.0 but there are differences. Acquiring 1.0 costs $300 even if you already have 2.0.

**Comment by nschneid** [comment]
Agreed, would be nice to have for 2.0 (or even 3.0, which is due out this year)! We started the project before 2.0 was released and decided it wasn't worth delaying the paper to update everything, but it would be great if somebody could work on that now.
